src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -265,36 +263,36 @@ public void bannedUserCanNotEnterTheChat() {
 
     switchToWindow(1);
     SpecialVersionPage chatWindow = new SpecialVersionPage().open();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "chatWindow" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)